.github/workflows/ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  security:
+    uses: dvsa/.github/.github/workflows/php-library-security.yml@main
+    with:
+      php-versions: "[\"7.4\"]"
+    secrets:
+      SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+
+  static-analysis:
+    uses: dvsa/.github/.github/workflows/php-library-static.yml@main
+    with:
+      php-version: '7.4'
+
+  tests:
+    uses: dvsa/.github/.github/workflows/php-library-tests.yml@main
+    with:
+      php-versions: "[\"7.4\"]"
+      fail-fast: false
```

Comment:
These are the wrong workflows - these are for libraries - we need the project equivalents.